### ISSUE TITLE
Issues with CED payment terminal not responding properly

### ISSUE THREAD

**Issue by djjd47130** [djjd47130/Cayan#38]
There are many issues, at random, when sending commands to the CED. For example, starting a new order, adding an item to Line Item Display, and even Cancel. 

In observation, when experiencing such issues in the POS app, the status shows "Offline" briefly at the same time. 

The most likely culprit is the worker thread which constantly monitors the current status of the CED. This same thread triggers events, which in turn are responsible for displaying its current screen in the POS app. This is not thread protected, and the thread needs to be isolated with its own HTTP client component.

**Comment by djjd47130** [comment]
Discovered one major flaw. In the past, the CED had an issue where it wouldn't respond ever to the HTTP requests, so at that time, I implemented a timeout. Mainly in the LID functions. However, that CED issue has since been fixed. But in either case, because there was a timeout, it was disconnecting before the CED could finish responding, or before some commands even made it to the CED at all. It was 900 milliseconds timeout. Now that the timeout has been removed, the problems have gone away, at least except for one occasions which is unknown why it failed. Likely related to the status monitoring thread, as mentioned before.
